.github/workflows/ci.yml

```diff
@@ -65,7 +65,8 @@ jobs:
 
       - name: Run Make install only for S. Aureus
         run:
-          make update_databases && df -h && make ${{ matrix.organism }}_all
+          SHORT_ORG=$(echo "${{ matrix.organism }}" | awk -F'_' '{print substr($1,1,1) $2}')
+          make update_databases && df -h && make ${SHORT_ORG}_all
```

Comment:
I suspect `SHORT_ORG` becomes empty here...

Reply:
What about 

1. Setting the variable with `export`
2. Adding an extra `&&`?

```yaml
          export SHORT_ORG=$(echo "${{ matrix.organism }}" | awk -F'_' '{print substr($1,1,1) $2}')
          && make update_databases && df -h && make ${SHORT_ORG}_all
```

... as I'm not 100% these two lines are interpreted as separate commands here otherwise, although honestrly I think they should :thinking:  ...

Reply:
You can see that this has been run:
```
make update_databases && df -h
```

Reply:
Yeah, this is not it.